PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanelConfigurable.java

```diff
@@ -63,18 +63,21 @@
     @NotNull
     private final Project myProject;
 
-    private SparkSubmissionContentPanel submissionPanel = new SparkSubmissionContentPanel();
+    protected SparkSubmissionContentPanel submissionPanel;
```

Comment:
Could you add an annotation `@Nullable`?

Reply:
this line has been reverted.